### ISSUE TITLE
Update localization

### ISSUE THREAD

**Issue by odie2** [rojer/a-s-minus#8]
Hi,

Nice child of that extension. Works much better.

However I would localize it to Polish, however available messages.json is even not half of phrases as I saw with [a453649](https://github.com/odie2/a-s-minus/commit/14abf71a6d5e21c7a242d9fdb23f0b158a739235).

Also why not translating extension name, as with [451a219](https://github.com/rojer/a-s-minus/commit/4995cf3e14869a5ce389d77ec9c12f286192ecb2)?
I know this is just extension, but it would be nicer to localize it, not translate, especially that could popularize Polish "zrzut ekranu" for "screenshot", as today more and more people go lazy and using international names.

Could you please update localization to give me possibility to translate each phrase to Polish?

Greetings

**Comment by rojer** [comment]
yes, i've been lazy on localization, using static strings in places where variables from the i18n files should  be used.
i keep that in the back of my mind as something to do, but you know how it goes... :)
let this issue be my reminder.
